ThirdParty/npm/protobufjs.js

```diff
@@ -0,0 +1,2 @@
+import * as protobuf from 'protobufjs/dist/minimal/protobuf.js';
```

Comment:
Is there a reason `import * as protouf from...` instead of `import protobuf...` here? Both seem to work with specs in my testing.

Reply:
The `import *` means give me everything where as `import protobuf` means "give me the default export only". In this case I'm pretty sure they are equivalent and it doesn't really matter.

Reply:
Also, https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import is a great reference for imports in general.